app/parse.py

```diff
@@ -14,5 +19,45 @@ class Course:
     course_type: CourseType
 
 
+def parse_single_course_card(course_soup: Tag) -> list[Course]:
+    name = course_soup.select_one(
+        "a.ProfessionCard_title__Zq5ZY"
+    ).text
+    short_description = course_soup.select_one(
+        "p.typography_landingTextMain__Rc8BD"
+    ).text
+    course_type = CourseType.PART_TIME
```

Comment:
You do not need to allocate this variable, you could pass `CourseType.PART_TIME` directly where needed.